src/MongoDB.Bson/IO/JsonWriter.cs

```diff
@@ -204,6 +209,23 @@ public override void WriteDateTime(long value)
                     _textWriter.Write("{{ \"$date\" : {0} }}", value);
                     break;
 
+                case JsonOutputMode.RelaxedExtendedJson:
+                    if (value >= BsonConstants.DateTimeMillisecondsStartEpoch &&
```

Comment:
I think this can simply be:

     if (value >= 0 &&

I don't see a need for a symbolic constant for zero.

Reply:
Done.